README.md

```diff
@@ -1,2 +1,9 @@
-# Hello Microverse
+# Hello Microverse- This is the first project in Microverse
  This is the first project in Microverse that helps us Micronauts to understand how to follow the general requirements of creating a project in Microverse. 
+ 
+
+Built With:
+    Html
+    CSS
+Author
+    Github: @adamteddychang
```

Comment:
- [ ] This is not a descriptive Readme. Please add a descriptive Readme to your project. Please use [this ](https://github.com/microverseinc/readme-template) template but remember to customize it to your project.

Reply:
Thanks for the information, do I copy the template and then edit it to fit my project?

Reply:
If so, what do I do with the set up instructions? Since it is a basic webpage that doesn't require set up, do I keep that section and leave it blank? Or should I delete it?